src/phoneNumber.ts

```diff
@@ -31,38 +31,28 @@ type PhoneNumberOptions = {
    * The minimum number to generate.
    * @default 10000000000
    */
-  min?: number
+  minLength?: number
   /**
    * The maximum number to generate.
    * @default 999999999999999
```

Comment:
**question**

Shouldn't that comment be changed to "maxLength ( @default=16 )" ? Same for minLength.

Reply:
Thank you for catching!